src/LINEBot.php

```diff
@@ -878,21 +879,51 @@ public function getNarrowcastProgress($requestId)
      */
     public function createAudienceGroupForUpdatingUserIds(
```

Comment:
I want to fix `updating` to `uploading`...

Reply:
Rename the method and create alias for it. And warn duplication.